css/global.css

```diff
@@ -781,6 +786,11 @@ header {
 	color: var(--color-white);
 }
 
+/* custom max width for hero content */
+.hero .wrapper {
```

Comment:
to match hero wrappers on all pages to have a shorter `max-width` than usual wrapper